extensions/BMO/template/en/default/pages/triage_owners.html.tmpl

```diff
@@ -102,6 +102,13 @@
           <tr>
             <th>[% r.product FILTER html %]</th>
             <th>Owner</th>
+            [% IF r.in_prod_security_group.defined %]
```

Comment:
Shouldn't we also check that the value is `'Yes'` here?

Reply:
> Shouldn't we also check that the value is `'Yes'` here?

No, this just determines if the column, with its header, will be displayed or not if the value is defined.

Reply:
Ah, makes sense.